packages/app-store/index.ts

```diff
@@ -28,6 +28,7 @@ const appStore = {
   exchangecalendar: () => import("./exchangecalendar"),
   facetime: () => import("./facetime"),
   sylapsvideo: () => import("./sylapsvideo"),
+  "zoho-bigin": () => import("./zoho-bigin"),
```

Comment:
It probably got removed during merge conflict resolution. https://github.com/calcom/cal.com/pull/8419/files#diff-8d0187f3e8930f70c4a3baae2905502d659e5bbadcf136e011d792c4ea7487e9L18